python/src/ecole/core/reward.cpp

```diff
@@ -144,6 +145,31 @@ void bind_submodule(py::module const& m) {
 
 		The difference in number of nodes is computed in between calls.
 		)");
+
+	auto solvingtime = py::class_<SolvingTime>(m, "SolvingTime", R"(
+		Solving time difference.
+
+		The reward is defined as the amount of time spent solving the instance since the previous state.
+        The solving time is maintained internally by SCIP and is specific to the operating system: it
+        includes presolving and time spent waiting on the agent.
+
+        N.B. Using this reward function will overwrite the `timing/clocktype` SCIP parameter.
+	)");
+	solvingtime.def(py::init<bool>(), py::arg("wall") = false, R"(
+		Constructor for SolvingTime.
+
+		Parameters
+		----------
+		wall : bool, optional
```

Comment:
No need to explicit the type, it is added by pybind because it knows the argument types.
Optional bool usually means, either a bool or `None`. Should `None` accpeted here?

Reply:
I'm not sure I understand what I should do here. You want me to remove the parameter description?

Reply:
I can't find any example of what parameter documentation looks like (there's also https://github.com/ds4dm/ecole/blob/b8ed4ff8c2f4858907e038b87886023dfabcfab7/python/src/ecole/core/observation.cpp#L193 but I suppose you would consider that incorrect too). What's the correct way to document a parameter then?

Also for me, and this is how I always use it in python documentation, optional means there is already a default value assigned to it (usually `None`, for booleans usually `False`).

Reply:
It's a detail, but I was pointing to the fact that `wall : bool, optional` could be replaced by `wall :`. It is redudant since pybind already knows the type and is able to add it to the docstring. Indeed `pseudo_candidates : bool` could be replaced with `pseudo_candidates :`.

As for optional, in the [Python typing system](https://docs.python.org/3/library/typing.html#typing.Optional), an `Optional[bool]` means _either a bool, or None_.
> Note that this is not the same concept as an optional argument, which is one that has a default.

Reply:
Ah ok, so the parameter documentation was fine, it was the just explicit type mention that was wrong. Got it. I fixed that.